includes/class-content-toc.php

```diff
@@ -118,11 +138,12 @@ public function shortcode( $atts, $content = null ) {
 				$item_text = strip_tags( $item_match_arr[1] );
 
 				// Add filter to allow custom TOC item markup
-				$toc_list .= apply_filters(
+				$items_html .= apply_filters(
```

Comment:
I'd say you don't need one here, gives people opportunity to break the functionality :)